conjur/errors.py

```diff
@@ -53,15 +54,15 @@ def __init__(self):
         super().__init__(self.message)
 
 class InvalidConfigurationException(Exception):
-    """ Exception for when configuration is invalid """
-    def __init__(self, message=""):
-        self.message = message
+    """ Exception for when configuration file (from .conjurrc) is in invalid format """
+    def __init__(self):
+        self.message = FETCH_CONFIGURATION_FAILURE_MESSAGE
```

Comment:
this is more flexible and I think we should keep the exceptions interface of adding message
```suggestion
    def __init__(self,message=FETCH_CONFIGURATION_FAILURE_MESSAGE):
        self.message = message
```